cookies/forms.py

```diff
@@ -279,3 +279,33 @@ class Meta:
 
 class ConceptEntityLinkForm(forms.Form):
     uri = forms.CharField(max_length=255, help_text='Manually enter a ConceptPower URI', label='URI')
+
+
+class UserAddCollectionForm(forms.Form):
```

Comment:
...and you'll also get quite a bit of the logic in this class for free, too (e.g. field declarations).